src/components/ha-label.ts

```diff
@@ -26,6 +26,7 @@ class HaLabel extends LitElement {
             0.15
           );
           --ha-label-background-opacity: 1;
+          border: 1px solid var(--divider-color);
```

Comment:
why `--divider-color` instead of `--outline-color`?

Reply:
Well,
1. In this PR a `border` is used instead of `outline` - since a parent element is clipping outline.
2. I got an impression that the `--divider-color` is mainly used for dividers/separators/etc, and the `--outline-color` is used mainly for outlines; also, a "plus" class for `ha-label` uses the `--divider-color`. So I thought that `--divider-color` fits better.
3. Currently both `--divider-color` & `--outline-color` have same values in a default theme - but ofc they could be different.
4. In the previous [PR](https://github.com/home-assistant/frontend/pull/22392) the `--outline-color` was used.

So, thinking about that - and probably we should:
-- either change here to `--outline-color` as well
-- or change to `--divider-color` in the 1st PR,
what do you think?

Reply:
Semantically outline is the same border here, especially since we already have `box-sizing: border-box`, so let's just use `--outline-color`